rasa/shared/nlu/training_data/formats/rasa_yaml.py

```diff
@@ -541,9 +549,7 @@ def _render_training_examples_as_objects(examples: List[Dict]) -> List[Dict]:
 
         def render(example: Dict) -> Dict:
             text = example[KEY_INTENT_TEXT]
-            example[KEY_INTENT_TEXT] = LiteralScalarString(
-                TrainingDataWriter.generate_string_item(text)
-            )
+            example[KEY_INTENT_TEXT] = LiteralScalarString(text + "\n")
```

Comment:
I've refactored the `TrainingDataWriter.generate_list_item` only for this line ([here](https://github.com/RasaHQ/rasa/pull/7761/files#diff-cd73e9fa63d1bf46bf4e4ce0c49e5f063894ddd9c6ad227a1ce06ff16f839d97)). It might make sense to move undo that change as well and remove the now unused `TrainingDataWriter.generate_string_item`?